addon-sidaneThreadmarks.xml

```diff
@@ -153,8 +153,12 @@
         </xen:if>
       </div>
     </div>
+    
+    <noscript>
+      <a href="{xen:link 'threads/threadmarks', $linkData}" rel="nofollow" class="threadmarksTrigger">{xen:phrase threadmarks}</a>
```

Comment:
Yeah that's the name of the variable being passed into the template by the pagination function.